### DISCUSSION TITLE
--concat option in ITSx

### DISCUSSION THREAD

**Discussion by blex-max** [blex-max/ecto_pipe#16] [Q&A]
In the ITSx manual, it presents the option to produce a file of concatenated ITS sequences - but doing so comes with a warning.

--concat {T or F} Saves a FASTA-file with concatenated ITS sequences (with 5.8S
pruned). Off (F) by default. Usage of this option is not endorsed,
as it will essentially produce chimeric ITS1+ITS2 sequences!

I don't understand this warning, can anyone shed any light on what they mean? Not sure why it prunes 5.8S either, makes it somewhat useless...

@catnipping ?

**Comment by catnipping** [comment]
ITSx normally produces a file with complete ITS and two separate files with ITS1 and ITS2 only. The structure of a complete ITS sequence is --ITS1--5.8S--ITS2 (plus possible partial SSU and LSU fragments hanging at the beginning and at the end of the sequence, respectively). My understanding is that the command --concat takes the two ITS fragments and links them together, generating an artificial ITS1--ITS2 structure, as it drops the 5.8S. The resulting sequence is "chimeric" in a way that it's not a real sequence. Having said that, I do not see the usefulness of this command for our purposes, and I'd leave it off as by default.
